cmd/pkg/update.go

```diff
@@ -97,9 +77,66 @@ ok pkg update my-package
 		true,
 		"Automatically migrate package configuration files to the latest version, if possible")
 
+	addRecursiveFlagToCmd(cmd, &flagRecursive, "Update")
+
 	return cmd
 }
 
+func createUpdateRecursiveFn(updater update.Updater, opts update.Options) RunRecursive {
+	return func(manifestPath string, manifestDir string, style lipgloss.Style) error {
+		fmt.Println()
+		fmt.Println(style.Render(fmt.Sprintf("Updating package manifest: %s", manifestPath)))
+		fmt.Println()
+
+		err := updateFromManifest(manifestPath, []string{}, manifestDir, updater, opts)
+		if err != nil {
+			return fmt.Errorf("installing from manifest %s: %w", manifestPath, err)
+		}
+
+		return nil
+	}
+}
+
+// TODO remove workingDirectpry?
+func updateFromManifest(manifestFile string, outputFolders []string, workingDirectory string, updater update.Updater, opts update.Options) error {
```

Comment:
Move workingDirectory to first argument - for me it then reads more naturally: directory first, then the file in that directory, then the output folders
```suggestion
func updateFromManifest(workingDirectory string, manifestFile string, outputFolders []string, updater update.Updater, opts update.Options) error {
```

Reply:
Fixed in 3d412e9f7107e6629ffbee31417c654522a127fb